### PR TITLE
Add: apng, png metadata extract feature

### LINKED ISSUE #2: not NullSafety
Could you please make the code NullSafety.

### PR DESCRIPTION
**Pull request by chanjungkim**
1. apng - frames +
2. png - width, height, gamma +
3. Dart 3 support

### CODE REVIEW

**Comment by chanjungkim** [comment]
@justkawal Please check this

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (justkawal) [inline .idea/workspace.xml:1]: Can you remove this file from the PR?

**Review by justkawal — APPROVED** [review]

File: .idea/workspace.xml (hunk 1 above)
Comment (chanjungkim) [reply .idea/workspace.xml:1]: Sorry I saw it.

**Comment by chanjungkim** [comment]
@justkawal Could you please release this version? It says it's not compatible with Dart 3.

**Comment by justkawal** [comment]
@chanjungkim Really, Sorry for the delay in publishing.
Version 1.0.3 is now available on pub.dev

Much appreciated for your PR.

**Pull request merged**